src/javascript/JContent/actions/deletePermanentlyAction.jsx

```diff
@@ -1,23 +1,30 @@
 import React from 'react';
 import {useNodeChecks} from '@jahia/data-helper';
+import {useSchemaFields} from './hooks/useSchemaFields';
 import PropTypes from 'prop-types';
 import {isMarkedForDeletion} from '../JContent.utils';
 import {useSelector} from 'react-redux';
 import {PATH_CONTENTS_ITSELF, PATH_FILES_ITSELF} from './actions.constants';
+import {findField} from '../JContent.utils';
 
 const checkAction = node => node.operationsSupport.markForDeletion &&
     isMarkedForDeletion(node) &&
-    node.aggregatedPublicationInfo.publicationStatus === 'NOT_PUBLISHED';
+    node.aggregatedPublicationInfo.publicationStatus === 'NOT_PUBLISHED' &&
+    (node.aggregatedPublicationInfo.existsInLive === undefined ? true : !node.aggregatedPublicationInfo.existsInLive);
 
 export const DeletePermanentlyActionComponent = ({context, render: Render, loading: Loading}) => {
     const {language} = useSelector(state => ({language: state.language}));
 
+    let types = useSchemaFields();
+    let existsInLiveField = types === undefined ? false : findField('existsInLive', types);
```

Comment:
This code should be done directly by useNodeInfo

Reply:
Ok